spinalcordtoolbox/cropping.py

```diff
@@ -38,9 +38,9 @@ def _get_max_value(input, dim):
             # If empty, return dim+1 (corresponds to the maximum of the given dimension, e.g. nx)
             if input is None:
                 return dim + 1
-            # If negative sign, return dim+1 if -1, dim if -2, dim-1 if -3, etc.
+            # If negative sign, return dim if -1, dim-1 if -2, dim-2 if -3, etc.
```

Comment:
To clarify (the comment is in contradiction with the code), i would write:
~~~
# If input is "-1", return maximum dimension (i.e. no change). If input is "-2", returns maximum dimension minus one, etc.
~~~
~~~